lib/tokens/token.js

```diff
@@ -40,28 +40,34 @@ module.exports = function (log, random, P, hkdf, Bundle, error) {
     this.algorithm = 'sha256'
     this.uid = details.uid || null
     this.lifetime = details.lifetime || Infinity
-    this.createdAt = optionallyOverrideCreatedAt(details.createdAt)
+    this.createdAt = details.createdAt || 0
```

Comment:
This token logic confuses me every time I look at it so apologies, but why do we need `|| 0` here? Won't `details.createdAt` always have been set how we want from the call to `optionallyOverrideCreatedAt` in `createNewToken`? Or are there some paths that lead to this constructor directly?